src/providers/log-provider.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Provider} from '@loopback/context';
+import {ParsedRequest} from '@loopback/core';
+
+export interface LogFn {
+  // tslint:disable-next-line:no-any
+  (req: ParsedRequest, args: any, result: any, startTime?: number[]): void;
+}
+
+export class LogProvider implements Provider<LogFn> {
+  constructor() {}
+
+  value(): LogFn {
+    return (
+      req: ParsedRequest,
+      // tslint:disable-next-line:no-any
+      args: any,
+      // tslint:disable-next-line:no-any
```

Comment:
You can replace all of the `no-any` comments with one above the exports:
`// tslint:disable:no-any`

Reply:
Normally, I'd say keep all the specific ones, but you've got it decorating a significant fraction of the code in this file, so might as well reduce your line count and cover it all in one go.